devtools/gha/unittest.py

```diff
@@ -0,0 +1,52 @@
+import importlib
+import inspect
+import json
+import os
+from pathlib import Path
+import sys
+import unittest
+
+source = importlib.util.find_spec('nutils').origin
+assert source.endswith(os.sep + '__init__.py')
+source = source[:-11]
+coverage = {}
```

Comment:
A comment explaining the values 0, 1 and 2 would be useful here.

Reply:
comment added